redfish-core/lib/oem/ibm/usb_code_update.hpp

```diff
@@ -0,0 +1,149 @@
+#pragma once

```

Comment:
Typically everything under redfish-core/lib/ is a route.. 
This looks more like a util https://github.com/openbmc/bmcweb/tree/b7028ebff16566762b71cdbc597c1244529d208a/redfish-core/include/utils

Reply:
oh well. 
This is a separate file so will carry downstream nicely.